extensions/blocks/recurring-payments/recurring-payments.php

```diff
@@ -9,11 +9,5 @@
 
 if ( ( defined( 'IS_WPCOM' ) && IS_WPCOM ) || Jetpack::is_active() ) {
 	require_once JETPACK__PLUGIN_DIR . '/modules/memberships/class-jetpack-memberships.php';
-
-	jetpack_register_block(
-		'jetpack/recurring-payments',
-		array(
-			'render_callback' => array( Jetpack_Memberships::get_instance(), 'render_button' ),
-		)
-	);
+	Jetpack_Memberships::get_instance();
```

Comment:
This isn't working on the frontend 😞 

Instead of a direct call to register the block, this moves it behind an action:
https://github.com/Automattic/jetpack/blob/11088101cea4ef93ab1e2eba00a24cbdecc486ff/modules/memberships/class-jetpack-memberships.php#L70-L73
https://github.com/Automattic/jetpack/blob/11088101cea4ef93ab1e2eba00a24cbdecc486ff/modules/memberships/class-jetpack-memberships.php#L104

That action runs here:

https://github.com/Automattic/jetpack/blob/11088101cea4ef93ab1e2eba00a24cbdecc486ff/class.jetpack-gutenberg.php#L370-L386

Which is before this file is included and the action is registered.

<details>

<summary>Some relevant places in code</summary>

https://github.com/Automattic/jetpack/blob/11088101cea4ef93ab1e2eba00a24cbdecc486ff/class.jetpack-gutenberg.php#L237-L271
https://github.com/Automattic/jetpack/blob/11088101cea4ef93ab1e2eba00a24cbdecc486ff/class.jetpack.php#L613
https://github.com/Automattic/jetpack/blob/11088101cea4ef93ab1e2eba00a24cbdecc486ff/class.jetpack-gutenberg.php#L660-L673

</details>

Reply:
Addressed this in 682fce9